RePlays.sln

```diff
@@ -1,20 +1,23 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31911.196
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34728.123
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RePlays", "RePlays.csproj", "{10BA385C-0777-4293-BED2-C85223FC2CF8}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RePlays", "RePlays.csproj", "{10BA385C-0777-4293-BED2-C85223FC2CF8}"
```

Comment:
Interesting that using a newer Visual Studio Version changes the project GUID. It does not seem like it affects anything really so I guess we can keep the changes.